authalligator_client/client.py

```diff
@@ -177,3 +182,60 @@ def query_account(
                 retry_in=result.account.retry_in,
             )
         return result.account
+
+    def delete_other_account_keys(
+        self,
+        provider,  # type: enums.ProviderType
+        username,  # type: str
+        account_key,  # type: str
+    ):
+        # type: (...) -> entities.DeleteOtherAccountKeysPayload
+        """Delete other account keys which have access to this account.
+
+        Args:
+            provider: the AuthAlligator provider this account is for. (should
+                be one of the ``ProviderType`` enum values)
+            username: the AuthAlligator-provided username for the account (this
+                will likely _not_ be the human-legible email/username)
+            account_key: the AuthAlligator-specific secret key that proves we
+                have access to the specific account
+
+        Returns:
+            Not much of anything, just a placeholder boolean. Raises an
+            exception for account errors.
+        """
```

Comment:
I'd probably do something like this:
```
Returns:
    DeleteOtherAccountKeysPayload on success.

Raises:
    AccountError in case of an account error.
```